src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const rows = sourceString
+    .trim()
+    .split(';')
+    .map((e) => e.split(':'));
+
+  rows.forEach((row) => {
+    const [property, value] = row;
+
+    if (property && value) {
```

Comment:
hmmm but we can have property = isBoolean and value = false and it won't work

Probably improve this moment but it isn't critical